src/theory/quantifiers/ce_guided_pbe.cpp

```diff
@@ -1516,7 +1517,7 @@ Node CegConjecturePbe::constructBestStringToConcat( std::vector< Node > strs,
                                                     UnifContext& x ) {
   Assert( !strs.empty() );
   // TODO
-  double r = (double)(rand())/((double)(RAND_MAX));
+  double r = (double)(RNG::getRNG().rand())/((double)(UINT64_MAX));
```

Comment:
This seems like a case for `pickDouble`.

Reply:
Ditto